lib/default-options.js

```diff
@@ -16,7 +16,8 @@ module.exports = function defaultOptions () {
             'assert.notDeepEqual(actual, expected, [message])',
             'assert.deepStrictEqual(actual, expected, [message])',
             'assert.notDeepStrictEqual(actual, expected, [message])'
-        ]
+        ],
+        additionalMethods: []
```

Comment:
At runtime, we are interested in just a few things:

1. Tracking calls for `t.plan()`
2. Tracking success/failure of each call
3. Magically extracting the optional `[message]` property and giving it to the callback. (This will make it easier to provide tap output where we list passed assertions).